client/lib/lib.rs

```diff
@@ -167,3 +176,436 @@ pub fn get_block(
 ) -> Result<JsonRpc> {
     RpcCall::new(maybe_rpc_id, node_address, verbose)?.get_block(maybe_block_hash)
 }
+
+/// Puts a deploy on a node.
+///
+/// * `maybe_rpc_id` is used for RPC-ID as required by the JSON-RPC specification, and is returned
+///   by the node in the corresponding response.  It must be either able to be parsed as a `u32` or
+///   empty.  If empty, a random ID will be assigned.
+/// * `node_address` identifies the network address of the target node's HTTP server, e.g.
+///   `"http://127.0.0.1:7777"`.
+/// * When `verbose` is `true`, the request will be printed to `stdout`.
+/// * `deploy` - Deploy arguments container struct holding options for this deploy. See
+///   `DeployStrParams`.
+/// * `session` - Session arguments container struct holding options for this deploy. See
+///   `SessionStrParams`.
+/// * `payment` - Payment arguments container struct holding options for this deploy. See
+///   `PaymentStrParams`.
+pub fn put_deploy(
+    maybe_rpc_id: &str,
+    node_address: &str,
+    verbose: bool,
+    deploy: DeployStrParams<'_>,
+    session: SessionStrParams<'_>,
+    payment: PaymentStrParams<'_>,
+) -> Result<JsonRpc> {
+    let deploy = Deploy::with_payment_and_session(
+        deploy.try_into()?,
+        payment.try_into()?,
+        session.try_into()?,
+    );
+    RpcCall::new(maybe_rpc_id, node_address, verbose)?.put_deploy(deploy)
+}
+
+/// Make a deploy and output to file or stdout.
+///
+/// * `maybe_rpc_id` is used for RPC-ID as required by the JSON-RPC specification, and is returned
+///   by the node in the corresponding response.  It must be either able to be parsed as a `u32` or
+///   empty.  If empty, a random ID will be assigned.
+/// * `node_address` identifies the network address of the target node's HTTP server, e.g.
+///   `"http://127.0.0.1:7777"`.
+/// * When `verbose` is `true`, the request will be printed to `stdout`.
```

Comment:
Copy-paste error - we should document `maybe_output_path` here.

Reply:
Thanks, done.